Library/Formula/openvpn.rb

```diff
@@ -13,10 +13,20 @@ class Openvpn < Formula
     sha256 "c2fb3da6a522f8f64330c238c9e5266cdea1d23e908113bb2b82eeb80f3e884d" => :mavericks
   end
 
+  option "with-pkcs11", "Enable pkcs11 support"
+
   depends_on "lzo"
   depends_on :tuntap if MacOS.version < :yosemite
   depends_on "openssl"
 
+  if build.with? "pkcs11"
+      depends_on "pkg-config" => :build
+      depends_on "autoconf" => :build
+      depends_on "automake" => :build
+      depends_on "libtool" => :build
+      depends_on "pkcs11-helper"
```

Comment:
We prefer badly named options and using `:optional` because of bugs in our dependency system.